es-space-mgmt/main/default/lwc/customerList/customerList.html

```diff
@@ -6,16 +6,18 @@
                 errors={errorMsg}
             ></c-error-panel>
         </template>
-        <ul class="slds-list_vertical slds-has-dividers_bottom-space">
-            <template for:each={customers} for:item="record">
-                <li class="slds-item" key={record.Id}>
-                    <c-customer-tile
-                        customer={record}
-                        object={sobject}
-                        oncustomerselect={publishSelect}
-                    ></c-customer-tile>
-                </li>
-            </template>
-        </ul>
+        <template if:true={customers}>
```

Comment:
Why did you modify the template?

Reply:
Because if `customers` is null (which it is, during page load, before the wire returns data), there is warning that says cannot iterate over undefined... So added this conditional rendering block

Reply:
Ok so instead of modifying the template which would require extra testing, lets intialize `customers` with an empty array.